views/default/js/mentions/autocomplete.js

```diff
@@ -11,6 +11,49 @@ define(function(require) {
 	var elgg = require('elgg');
 	var callback;
 
+ /**
+  *  positionMentionPopup()
+	*
+	*  get position in pixels of the editor's cursor and place the mention popup relative to it.
+	**/
+
+	var positionMentionPopup = function() {
+		if (typeof tinyMCE.activeEditor != "undefined")
```

Comment:
All editor specific code should go to one of these files: https://github.com/Elgg/mentions/tree/master/views/default/js/mentions/editors

We do not want to load tinymce specific code on every page load in sites that aren't even using tinymce.